core/src/main/scala/akka/persistence/r2dbc/internal/postgres/PostgresSnapshotDao.scala

```diff
@@ -168,40 +207,42 @@ private[r2dbc] class PostgresSnapshotDao(settings: R2dbcSettings, connectionFact
   }
 
   def store(serializedRow: SerializedSnapshotRow): Future[Unit] = {
-    val entityType = PersistenceId.extractEntityType(serializedRow.persistenceId)
-    val slice = persistenceExt.sliceForPersistenceId(serializedRow.persistenceId)
+    queryDao.timestampOfEvent(serializedRow.persistenceId, serializedRow.seqNr).flatMap { eventTimestamp =>
```

Comment:
some overhead when storing the snapshot because I want to use the same timestamp (offset) as the event so that Projection OffsetStore will handle the snapshot event equal to the corresponding ordinary event.

Reply:
Could do it when reading instead, but then more overhead for each query.